auacm/app/modules/submission_manager/views.py

```diff
@@ -69,3 +69,44 @@ def submit():
     return serve_response({
         'submissionId': attempt.job
     })
+
+
+@app.route('/api/submit/job/<int:job_id>')
```

Comment:
Similar to the note below, I think we should use a different route here to remain RESTful. Since we're accessing a particular submissions, we should be doing something more like `/api/submission/<int:job>`, since we're accessing a specific submission.